### ISSUE TITLE
Repo is down Error 522

### ISSUE THREAD

**Issue by maxamula** [akemin-dayo/AppSync#215]
Cannot install appsync via Cydia or Sileo
![IMG_7411E672CE95-1](https://github.com/akemin-dayo/AppSync/assets/78308778/7afb0a27-fc7f-4f2f-b004-c90fd14fe8c0)

**Comment by Rustler69** [comment]
Still down on July 19th
I've tried to contact Karen, got no response.
Seems like she went totally offline for some reason.
Someone should really mirror her repo with at least appsync

**Comment by akemin-dayo** [comment]
I have unfortunately still been unable to get into contact with the person handling the host hardware for the specific VM that hosts my Cydia/APT repo (as well as most of my web-facing services).

I will try to get a temporary instance of my repo up and running, though some packages (as well as repo depictions in general) may not be available.

Unfortunately due to the nature of current circumstances, I am unable to simply just spin up another VM instance from a filesystem backup as I otherwise normally would, hence the continued downtime. This is the first time in over 10 years(!) of this server's existence that anything like this has happened, but I have plans going forward to prevent extended downtime like this from ever happening again.

(※ https://twitter.com/akemin_dayo/status/1814227264185504213)

**Comment by Rustler69** [comment]
Whoa, I started to worry that sth bad happened to You personally, so glad to hear from you :)
When you have some time to spare (and whilst on bringing everything back online), can you also update AppSync to work with latest Dopamine and iOS 16.6? I had no chance to trying it yet (due to outage of your repo - such bad timing ;) and I'm still running iOS14 on another device to have a rootful JB), but I will have to update to iOS 16 soon, as more and more apps cease to work (due to devs dropping support to 14). And i've seen some posts about AppSync refusing to coop with Dopamine 2.0.

**Comment by akemin-dayo** [comment]
Yes, Dopamine 2.0 (and general rootless jailbreak support) will be added.

**Comment by b-risk** [comment]
anything i can do to get appsync installed yet?

**Comment by Rustler69** [comment]
Do You have a rough idea when your repo (and obviously the long awaited AppSync v.112 ;) will be brought alive again..? There are legions of us, jailbreakers, longing for it really bad ;)
Terribly sorry for urging, but only myself have now two devices that require your tiny but so terribly necessary .deb package to be 100% functional :)
Alternatively, is there any chance You could drop the sole file here, on github? That would be most appreciated :)

**Comment by dmatora** [comment]
does anyone know any other repo that has it?

**Comment by b-risk** [comment]
> does anyone know any other repo that has it?

I was asking the same question weeks ago here but never got an answer, but I finally found a repo called "Skyglow" that seems to have it.

**Comment by Rustler69** [comment]
> > does anyone know any other repo that has it?
> 
> I was asking the same question weeks ago here but never got an answer, but I finally found a repo called "Skyglow" that seems to have it.

Aforementioned repo does exist, but doesn't have AppSync in any given version, even outdated.
Perhaps you had a different repo in mind?

**Comment by b-risk** [comment]
> > > does anyone know any other repo that has it?
> > 
> > 
> > I was asking the same question weeks ago here but never got an answer, but I finally found a repo called "Skyglow" that seems to have it.
> 
> Aforementioned repo does exist, but doesn't have AppSync in any given version, even outdated. Perhaps you had a different repo in mind?

Sorry for not responding I forgot this existed!
That's so weird, that's was it says on Cydia for me
![IMG_20240822_121754_459](https://github.com/user-attachments/assets/af60ea4f-e32a-4e02-8888-08ae947d3fee)
The full link to the repo I have that has it is cydia.skyglow.es
There's also another repo that has it but it has *argg ye matey* on it so I probably won't mention it's name but you can probably find it online or by searching for another repo with appsync, use it at your own risk though

**Comment by Rustler69** [comment]
That's why You see it - you're on rootful jb, aren't you? And by the looks of cydia, quite ancient iOS..? XD Rootless version of appsync (at least rumor has it) was the latest - 112, and Zebra on rootless filters out any rootful tweaks - that's why I cannot see it in this repo. Unfortunately, any earlier than 112 will not do in case of iOS16 - if that, that is...

**Comment by b-risk** [comment]
> That's why You see it - you're on rootful jb, aren't you? And by the looks of cydia, quite ancient iOS..? XD Rootless version of appsync (at least rumor has it) was the latest - 112, and Zebra on rootless filters out any rootful tweaks - that's why I cannot see it in this repo. Unfortunately, any earlier than 112 will not do in case of iOS16 - if that, that is...
> 
> All in all, we just have to wait for Karen to fix her part, as she mentioned...

I misunderstood, I found this from the scene of untethered iOS 6 tweaking/jailbreaking and thought that is what it is mainly used for, now it makes sense to me why it wouldn't show because sky-glow is only for older iOS versions and only has older tweaks. Sorry for wasting your guys time.

**Comment by Rustler69** [comment]
You never waste time whilst doing research, young padawan ;-}} good work with digging it out, I've never come across any 3rd party repo that would host appsync in any version, props to you :)

**Comment by akemin-dayo** [comment]
【Update regarding iOS repo downtime】
Apologies for the continued delay in the restoration of my iOS Cydia/APT repo services.

I expect to be able to partially get it back up sometime in the first half of September.

The situation is unfortunately far more complicated than I have publicly mentioned.

Please understand and respect that I will not go to into details beyond this, but I am at this point quite concerned for the wellbeing of a friend of mine who has provided the server hardware that the VM running these services is located on.

Normally, I'd instantiate a new VM node from a filesystem backup, but for complicated reasons (many of which are my own fault), that is not currently possible.

Also, I can't believe I have to say this, but please stop sending me threats of violence and SA. That helps no one.

(※ https://twitter.com/akemin_dayo/status/1829342520800932249)

**Comment by maitto** [comment]
Hi @akemin-dayo, any updates on this?

**Comment by Rustler69** [comment]
Hi Karen, have you got any updates for us..? :)

**Comment by BlackCatOfficialytb** [comment]
Just host in GITHUB PAGES, you can actually do that, make the alternative repo is better than finding the way to start the main repo 💀

**Comment by Rustler69** [comment]
Yay, she responded! I've got the .deb files

**Comment by jorgee2001** [comment]
@Rustler69 can you share me the .deb files? please

**Comment by Nixuge** [comment]
Any news about this? It’s now been a year since the last repo update, is it permanently dead or are you still planning to get it back up at some point?

**Comment by LilData777** [comment]
@Nixuge Ya know, I see a release on this repo on Github. It is a Janurary one. It contains deb files. I suggest looking into that. I plan to.

**Comment by LilData777** [comment]
@jorgee2001 I think the deb files is the release on Github. Check that.

**Comment by kocoman1** [comment]
Is there any way to install appsync unified?  I getting 522 error thx
